core/src/mindustry/world/blocks/campaign/LaunchPad.java

```diff
@@ -175,6 +176,7 @@ public void buildConfiguration(Table table){
 
         Seq<ItemStack> stacks = new Seq<>();
         transient Interval in = new Interval();
+        private TextureRegion region;
```

Comment:
Regions cannot be serialized. Save a reference to the `Block` instead, cast it to a `LaunchPad` and use its region. Don't make this private, there's no reason to use getters here.

Reply:
You suggested a more complex and costly variant, because while it searches for tiles, checks LaunchPad it or no and looks for podRegion in it, it will take a lot of unnecessary MB

Reply:
Maybe I misunderstood you?

Reply:
No. `Block block;` and `((LaunchPad)block).region` is not any more memory intensive.

You also need to check for block/region being null, because older versions won't have it serialized.

Reply:
This work but need fix 
![image](https://user-images.githubusercontent.com/61590217/103431951-6510e000-4be9-11eb-990a-91a95957f175.png)